Makefile

```diff
@@ -1,10 +1,17 @@
 .PHONY: lint test vet
 
+# The 'test' target executes Go tests for all packages
```

Comment:
I'd just skip all of these comments, they don't add anything that isn't obvious from the Makefile target body.